metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-rest-env.xml

```diff
@@ -90,17 +90,11 @@
     </property>
     <property>
         <name>metron_temp_grok_path</name>
-        <description>Temporary local file path where grok patterns are written during testing</description>
-        <value>./patterns/temp</value>
+        <description>Temporary HDFS file path where grok patterns are written during testing</description>
```

Comment:
So, we are creating a tmp sensor, to put the patterns in?

Reply:
No.  The parse and init methods are called directly on the configured MessageParser class so there is no need to persist a tmp parser config or anything like that.

Reply:
no, what I mean is the structure of /patterns is /patterns/sensor.
Pre 777 sensor is a single grok file with parser name.
Post it is a directory with the parser name, containing one or more grok files

So putting something in /patterns  is kind of creating a faux parser area.

Reply:
If you think it would make it easier for 777 we can deafult this temp path to something else.  It could be any path that makes sense. 

When we switch to using a directory containing one or more grok files this will likely need to be updated anyways.  For now this is meant to fix the UI which is currently broken.